plugins/modules/ndo_fabric_span_session.py

```diff
@@ -0,0 +1,705 @@
+        - The value must be in the range 1 - 1023.
+        - Defaults to 1 when unset during creation.
+        type: int
+      dscp:
+        description:
+        - The DSCP value for sending the monitored SPAN Session packets.
+        - Defaults to C(unspecified) when unset during creation.
+        choices:
+          - af11
+          - af12
+          - af13
+          - af21
+          - af22
+          - af23
+          - af31
+          - af32
+          - af33
+          - af41
+          - af42
+          - af43
+          - cs0
+          - cs1
+          - cs2
+          - cs3
+          - cs4
+          - cs5
+          - cs6
+          - cs7
+          - expedited_forwarding
+          - voice_admit
+          - unspecified
+        type: str
+  destination_port:
+    description:
+    - The destination port configuration group.
+    - This parameter or O(destination_epg) or O(destination_port_channel) is required when creating the SPAN Session.
+    type: dict
+    suboptions:
+      port_uuid:
+        description:
+        - The UUID of the destination port to use for the SPAN Session.
+        - This parameter or O(destination_port.port) is required.
+        type: str
+      port:
+        description:
+        - The destination port to use for the SPAN Session.
+        - This parameter or O(destination_port.port_uuid) is required.
+        type: dict
+        suboptions:
+          node:
+            description:
+            - The Node ID to use for the SPAN Session.
+            type: int
+            required: true
+          path:
+            description:
+            - The Node Ethernet interface to use for the SPAN Session.
+            type: str
+            required: true
+  destination_port_channel:
+    description:
+    - The destination port channel configuration group.
+    - This parameter or O(destination_epg) or O(destination_port) is required when creating the SPAN Session.
+    type: dict
+    suboptions:
+      port_channel_uuid:
+        description:
+        - The UUID of the destination port channel to use for the SPAN Session.
+        - This parameter or O(destination_port_channel.port_channel) is required.
+        type: str
+      port_channel:
+        description:
+        - The destination port channel to use for the SPAN Session.
+        - This parameter or O(destination_port_channel.port_channel_uuid) is required.
+        type: dict
+        suboptions:
+          name:
+            description:
+            - The name of the destination port channel.
+            type: str
+            required: true
+          template:
+            description:
+            - The name of the template that contains the destination port channel.
+            - This parameter or O(destination_port_channel.port_channel.template_id) is required.
+            type: str
+          template_id:
+            description:
+            - The ID of the  that contains the destination port channel.
```

Comment:
This sentence is incomplete, also extra space between the and that

Reply:
Done!